### DISCUSSION TITLE
Q: Continuous Batching without Turbomind?

### DISCUSSION THREAD

**Discussion by SinanAkkoyun** [InternLM/lmdeploy#2029] [Q&A] [answered]
Hello!
I wanted to ask: Do models like `Deepseek Coder v2`, which don't have turbomind support, also have continuous batching support or how does it work with them?

**Comment by zhyncs** [comment, marked as answer]
PyTorch Engine also supports continuous batching :) cc @grimoire

**Reply by SinanAkkoyun** [reply]
Great, thank you! :)
